### PR TITLE
Feature-Aaron:welcome-email-template

### PR DESCRIPTION
**Pull request by twikista**
added welcome email template

### CODE REVIEW

File: app/api/Templates/welcomeEmailTemplate.handlebars [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    apps --}}
+    <meta name="x-apple-disable-message-reformatting" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&family=Pacifico&family=Roboto+Condensed:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Welcome Email Template</title>
+     {{!-- code below ensurs that PNG images display at the right size in Outlook
```
Comment (Remi-dee) [inline app/api/Templates/welcomeEmailTemplate.handlebars:20]: You can take out the comments, might not be necessary.

File: app/api/Templates/welcomeEmailTemplate.handlebars [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            talents face in their search for jobs that truly match their skills.
+          </p>
+          <p style="padding: 10px 0">
+            As a {{role}}, our proprietary layered talent matching system puts
+            you in total control of the opportunities you're matched with, while
+            ensuring you grow your career.
+          </p>
+        </td>
+      </tr>
+      {{!-- profile action --}}
```
Comment (Remi-dee) [inline app/api/Templates/welcomeEmailTemplate.handlebars:95]: Here as well

**Review by Remi-dee — CHANGES_REQUESTED** [review]

**Comment by twikista** [comment]
comments removed

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Codes shouldn't be commented and sent to the PR

**Comment by twikista** [comment]
> Codes shouldn't be commented and sent to the PR

All comments removed from code

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Use the corrections on the other PR to fix the concerns in this one as well

**Comment by twikista** [comment]
> Use the corrections on the other PR to fix the concerns in this one as well

corrections implemeted
@annonymousauthority

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**